CRM/Core/Module.php

```diff
@@ -55,7 +66,7 @@ public static function getAll($fresh = FALSE) {
     if ($fresh || !is_array($result)) {
       $result = CRM_Extension_System::singleton()->getMapper()->getModules();
       // pseudo-module for core
-      $result[] = new CRM_Core_Module('civicrm', TRUE);
+      $result[] = new CRM_Core_Module('civicrm', TRUE, ts('CiviCRM'));
```

Comment:
@totten in #27389 I called it `ts('CiviCRM Core')`. I guess it's fine either way but I chose that label for clarity since a lot of these strings will contain "CiviCRM" and the unadorned version might get lost in the crowd when visually scanning the list.

Reply:
@colemanw OK, that works for me. Updated.